cloud/services/domains.go

```diff
@@ -37,14 +40,19 @@ func EnsureDNSEntries(ctx context.Context, mscope *scope.MachineScope, operation
 		return nil
 	}
 
+	// Verify that the ClusterNetworkSpec is set
+	if cmp.Equal(mscope.LinodeMachine.Status.ClusterNetworkSpec, v1alpha2.NetworkSpec{}) {
```

Comment:
[cmp should only be used for tests since it can panic if it can't compare values](https://pkg.go.dev/github.com/google/go-cmp/cmp)

Reply:
fixed with [this commit](https://github.com/linode/cluster-api-provider-linode/pull/433/commits/c44ec02ba3d909b107d85cb13bacd871cdb11277)

Reply:
awesome, thanks! 🙏